src/api/app/controllers/webui/repositories_controller.rb

```diff
@@ -165,6 +165,8 @@ def change_flag
     locals[:table_id] = "flag_table_#{flag_type}"
 
     render partial: 'webui/shared/repositories_flag_table', locals: locals
+  rescue ActiveRecord::RecordInvalid
```

Comment:
This place to catch this is where the change is happening. On `follow_change_flag_command` a couple of lines below...

Reply:
Changed